vizro-core/src/vizro/actions/export_data_action.py

```diff
@@ -23,7 +23,7 @@ def export_data(
     """Exports visible data of target charts/components on page after being triggered.
 
     Args:
-        targets: List of target component ids to download data from. Defaults to None.
+        targets: List of target component ids to download data from. Defaults to `None`.
```

Comment:
Just reminded myself of how mkdocstring handles defaults. It is unfortunately a little confusing...

For pydantic models: we need to mention the default (e..g defaults to `None`), because the default is not recognized

For python functions (such as actions), this is actually automatically recognized (i.e. it will know from the type hint)

While being (for the time being) two different approaches, I would suggest that we remove the "defaults to XY" from all actions.

Wdyt?

Reply:
Given that it seems quite confusing and people would then have to remember when to add it or not, I would actually just leave it as is and always provide it? Currently it's consistently added to the docstrings everywhere and this seems like a simpler rule to remember until we have consolidated everything?

Reply:
Hm ok, then let's leave it